src/environments/assets.ts

```diff
@@ -1,4 +1,5 @@
 export const assets = [
+  'assets/release-notes-1.30.0-alpha.2.json',
```

Comment:
@OrlinVasilev did you add this manually? or it was added automatically by krel?

Reply:
Nvm, I reviewed [other PR](https://github.com/kubernetes-sigs/release-notes/pull/212/files) and this was added by `krel`.